src/v3_1/paths.rs

```diff
@@ -0,0 +1,177 @@
+use std::marker::PhantomData;
+
+use crate::{util::*, v3_1::*};
+use indexmap::IndexMap;
+use serde::{Deserialize, Deserializer, Serialize};
+
+/// Describes the operations available on a single path.
+/// A Path Item MAY be empty, due to ACL constraints.
+/// The path itself is still exposed to the documentation
+/// viewer but they will not know which operations and
+/// parameters are available.
+#[derive(Debug, Clone, Serialize, Deserialize, Default, PartialEq)]
+pub struct PathItem {
+    /// Allows for a referenced definition of this path item. The referenced
+    /// structure MUST be in the form of a Path Item Object.  In case a Path
+    /// Item Object field appears both in the defined object and the referenced
+    /// object, the behavior is undefined. See the rules for resolving Relative
+    /// References.
+    #[serde(rename = "$ref", skip_serializing_if = "Option::is_none")]
+    reference: Option<String>,
```

Comment:
please make this field public, so we continue to be able to use struct literal syntax for these!